### DISCUSSION TITLE
useId 在 SSR 模式下的多次客户端请求会和客户端渲染的不一致

### DISCUSSION THREAD

**Discussion by yoyo837** [youzan/vant#12674] [Q&A]
服务端的 id 在递增，客户端的始终从 0 开的的，
比如 van-tabs-id

不知道如何提供 SSR 的复现，所以提到了这里 @chenjiahan @inottn

**Comment by inottn** [comment]
和这个问题类似吗 https://github.com/youzan/vant/issues/12548

**Reply by yoyo837** [reply]
嗯，类似，但我没有使用 nuxt，我只有 vue + vant (astro ssg), 而且  userId 来自 vant packages/vant/src/composables/use-id.ts
